### PR TITLE
docs(tasks/log-level): add MQTT X CLI connect EMQX

### PR DESCRIPTION
**Pull request by Gala-R**

### CODE REVIEW

File: docs/en_US/tasks/configure-emqx-log-level.md [hunk 1]
```diff
@@ -92,27 +88,28 @@ $ kubectl get emqxenterprises
...
+[MQTT X CLI](https://mqttx.app/cli) is an open source MQTT 5.0 command line client tool, designed to help developers to more Quickly develop and debug MQTT services and applications.
 
-Click the button to create a new connection on the MQTT X page, and configure the EMQX cluster node information as shown in the figure. After configuring the connection information, click the connect button to connect to the EMQX cluster:
+- Subscribe to messages using MQTT X CLI
 
-<img src="./assets/configure-log-level/mqtt-connected.png" style="zoom:50%;" />
-
-Then click the Subscribe button to create a new subscription, as shown in the figure, MQTT X has successfully connected to the EMQX cluster and successfully created the subscription:
+```bash
+mqttx sub -h ${loadBalancer_ip} -p 1883 -t "test/topic"
```
Comment (Rory-Z) [inline docs/en_US/tasks/configure-emqx-log-level.md:101]: I think in this document, `mqttx conn` is ok, doesn't need pub / sub

File: docs/en_US/tasks/configure-emqx-log-level.md [hunk 2]
```diff
@@ -92,27 +88,28 @@ $ kubectl get emqxenterprises
...
+```
 
-<img src="./assets/configure-log-level/mqtt-pub.png" style="zoom:50%;" />
+> `${loadBalancer}` is the LoadBalancer IP corresponding to EMQX Service.
 
-Use the command line to view EMQX cluster log information
+- Use the command line to view EMQX cluster log information
 
 ```bash
 kubectl logs emqx-core-0 -c emqx
```
Comment (Rory-Z) [inline docs/en_US/tasks/configure-emqx-log-level.md:115]: 删除图片，输出 connect 报文相关的 log

File: docs/zh_CN/tasks/configure-emqx-log-level.md [hunk 3]
```diff
@@ -11,11 +11,9 @@
 :::: tabs type:card
 ::: tab v2alpha1
 
-EMQX CRD 支持使用 `.spec.bootstrapConfig` 来配置 EMQX 集群日志等级，bootstrapConfig 的配置可以参考文档：[bootstrapConfig](https://www.emqx.io/docs/zh/v5.0/admin/cfg.html)。这个字段只允许在创建 EMQX 集群的时候配置，不支持更新。
+EMQX CRD 支持使用 `.spec.bootstrapConfig` 来配置 EMQX 集群日志等级，bootstrapConfig 的配置可以参考文档：[bootstrapConfig](https://www.emqx.io/docs/zh/v5.0/admin/cfg.html)。
 
-:::tip
-如果在创建 EMQX 之后需要修改集群日志等级，请通过 EMQX Dashboard 进行修改。
-:::
+> 这个字段只允许在创建 EMQX 集群的时候配置，不支持更新。如果在创建 EMQX 之后需要修改集群日志等级，请通过 EMQX Dashboard 进行修改。
```
Comment (Rory-Z) [inline docs/zh_CN/tasks/configure-emqx-log-level.md:16]: Please use :::tip

File: docs/zh_CN/tasks/configure-emqx-log-level.md [hunk 4]
```diff
@@ -1,123 +1,134 @@
...
 
 -  使用命令行查看 EMQX 集群日志信息
 
 ```bash
-kubectl logs emqx-core-0 -c emqx
+$ kubectl logs emqx-core-0 -c emqx
 ```
 
-输出如下图所示：
+输出类似于：
```
Comment (Rory-Z) [inline docs/zh_CN/tasks/configure-emqx-log-level.md:127]:
```suggestion
可以获取到类似如下的打印，这意味着 EMQX 收到了一个来自客户端的 CONNECT 报文，并向客户端回复了 CONNACK 报文：
```

File: docs/zh_CN/tasks/configure-emqx-log-level.md [hunk 5]
```diff
@@ -1,123 +1,134 @@
...
 
-<img src="./assets/configure-log-level/emqx-debug-log.png" style="zoom:33%;" />
+```bash
+2023-04-17T09:11:35.993031+00:00 [debug] msg: mqtt_packet_received, mfa: emqx_channel:handle_in/2, line: 360, peername: 218.190.230.144:59457, clientid: mqttx_322680d9, packet: CONNECT(Q0, R0, D0, ClientId=mqttx_322680d9, ProtoName=MQTT, ProtoVsn=5, CleanStart=true, KeepAlive=30, Username=undefined, Password=), tag: MQTT
+2023-04-17T09:11:35.997066+00:00 [debug] msg: mqtt_packet_sent, mfa: emqx_connection:serialize_and_inc_stats_fun/1, line: 872, peername: 218.190.230.144:59457, clientid: mqttx_322680d9, packet: CONNACK(Q0, R0, D0, AckFlags=0, ReasonCode=0), tag: MQTT
+```
 
-从图中可以看到刚才使用 MQTT 连接 EMQX 集群建立连接以及发送消息的 debug 日志信息。
+从输出的日志信息中可以看到刚才使用 MQTT X CLI 连接 EMQX 集群的 CONNECT 报文信息。
```
Comment (Rory-Z) [inline docs/zh_CN/tasks/configure-emqx-log-level.md:134]: 删除这一行

File: docs/zh_CN/tasks/configure-emqx-log-level.md [hunk 6]
```diff
@@ -1,123 +1,133 @@
-# 修改 EMQX 日志等级
+# 修改 EMQX 日志等级·
```
[the remark this answers is not in the document]
Comment (Gala-R) [reply docs/zh_CN/tasks/configure-emqx-log-level.md]: > Title 不需要句号

Done

**Review by Rory-Z — APPROVED** [review]

**Pull request merged**